doc/cli_vs_yum.rst

```diff
@@ -219,7 +219,7 @@ During its depsolving phase, YUM outputs lines similar to::
 
 DNF does not output information like this. The technical reason is that depsolver below DNF always considers all dependencies for update candidates and the output would be very long. Secondly, even in YUM this output gets confusing very quickly especially for large transactions and so does more harm than good.
 
-See the the related `Fedora bug 1044999
+See the related `Fedora bug 1044999
```

Comment:
It would be nice to have this and the below "the" deduplication in a commit separate from the plugin list changes.

I'm not sure how strict we are about having separate commits for unrelated changes so I don't know if I'm supposed to ask you to fix it or leave this as an advice for the future :smile: .